.well-known/web.config

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<configuration>
+  <system.webServer>
+    <rewrite>
+      <rules>
+        <!-- We'll clear the rules so parent rules don't take effect, to ensure there are no redirects for the sake of apple-app-site-association -->
+        <clear />
+        <rule name="apple-app-site-association">
+          <match url="^apple-app-site-association$" />
+          <action type="Rewrite" url="apple-app-site-association.json" />
+        </rule>
+      </rules>
+      <outboundRules>
+        <!-- We're going to force just application/json with no charset according to the Apple 'docs' -->
+        <rule name="Add mime type for apple-app-site-association">
+          <match serverVariable="RESPONSE_Content-Type" pattern='.' />
```

Comment:
Does this impact the Android assetlinks.json that's now in the same folder?

Reply:
No, because of the condition on line 18.